code/modules/surgery/organs/organ_external.dm

```diff
@@ -847,28 +852,27 @@ Note that amputating the affected organ does in fact remove the infection from t
 	status &= ~ORGAN_BROKEN
 	return 1
 
-// I put these two next to each other to highlight that both exist. This should likely be resolved.
-/obj/item/organ/external/robotize()
+/obj/item/organ/external/robotize(company, make_tough = 0, convert_all = 1)
```

Comment:
I have a hunch that convert_all defaulting value being 1 will mean heads (being attached to a parent bodypart [chest]) are converted to the brand of the chest. Setting this to 0 should fix the problem, though IDK what problems having it at 0 (though surgery calls this proc with convert_all set to 0, anyway). @Fox-McCloud 

IDK what the deal with the issue is myself but I can't see anything else that could possibly be causing the problem.